README.md

```diff
@@ -50,24 +50,23 @@
 
 ## Introduction
 
-**GreptimeDB** is an open-source time-series database focusing on efficiency, scalability, and analytical capabilities.
-Designed to work on infrastructure of the cloud era, GreptimeDB benefits users with its elasticity and commodity storage, offering a fast and cost-effective **alternative to InfluxDB** and a **long-term storage for Prometheus**.
+**GreptimeDB** is an open-source unified time-series database for **metrics**, **events**, and **logs**. You can gain real-time insights from Edge to Cloud at any scale.
 
 ## Why GreptimeDB
 
 Our core developers have been building time-series data platforms for years. Based on our best-practices, GreptimeDB is born to give you:
 
-* **Easy horizontal scaling**
+* **Unified all kinds of time series**
 
-  Seamless scalability from a standalone binary at edge to a robust, highly available distributed cluster in cloud, with a transparent experience for both developers and administrators.
+  GreptimeDB treats all time series as contextual events with timestamp, and thus unifies the processing of metrics and events. It supports analyzing metrics and events with SQL and PromQL, and doing streaming with continuous aggregation.
 
-* **Analyzing time-series data**
+* **Cloud-Edge collaboration**
 
-  Query your time-series data with SQL and PromQL. Use Python scripts to facilitate complex analytical tasks.
+  GreptimeDB can be deployed on ARM architecture-compatible Android/Linux systems as well as cloud environments from various vendors. Both sides run the same software, providing identical APIs and control plane, so your application can run at the edge or on the cloud without modification.
 
-* **Cloud-native distributed database**
+* **Easy horizontal scaling**
```

Comment:
Keep the title not changing. I think `cloud-native` is more proper here.

Reply:
```suggestion
* **Cloud-native distributed database**
```